mvn-extension/src/main/java/org/arquillian/smart/testing/mvn/ext/SmartTestingMavenConfigurer.java

```diff
@@ -28,7 +29,8 @@
     hint = "smart-testing")
 class SmartTestingMavenConfigurer extends AbstractMavenLifecycleParticipant {
 
-    private static final Logger logger = Logger.getLogger();
+    @Requirement
+    private org.codehaus.plexus.logging.Logger LOGGER;
```

Comment:
If it's not static then it shouldn't be uppercase. Also we can import it as usual, no need for fully qualified package name here.